enrolments/validators.py

```diff
@@ -24,21 +24,21 @@ def validate_class_in_session(class_obj, session):
         )
 
 
-def validate_enrolment(enrolment):
-    validate_class_in_session(enrolment.preferred_class, enrolment.session)
-    validate_class_in_session(enrolment.enrolled_class, enrolment.session)
-
-
-def validate_class_in_session(class_obj, session):
-    if class_obj.session != session:
+def validate_enrolment_in_session(session, family):
+    Enrolment = apps.get_model("enrolments", "Enrolment")
+    family_enrolments_per_session = Enrolment.objects.filter(
```

Comment:
nit: what are your thoughts on renaming `family_enrolments_per_session` to `family_enrolments_in_session`. Only suggesting this bc the variable is the enrolments for a specific session and not all the sessions.

Reply:
yes! good idea